app/src/main/java/ch/epfl/sdp/blindly/profile/Profile1.kt

```diff
@@ -13,7 +14,7 @@ class Profile1 : AppCompatActivity() {
     }
 
     fun startProfile2(view: View) {
```

Comment:
Function name can be updated since it doesn't actually start Profile 2 right now, but I don't think we should start with asking Permissions right away in the profile. Similar apps seem to do it at the end of profile preferences.

Reply:
We discussed it on whatsapp if we all agree to place it at the end it can be easily done

Reply:
Exactly!